packages/control-property-editor/development/server.js

```diff
@@ -0,0 +1,169 @@
+const { join } = require('path');
```

Comment:
Could you please use the `preview-middleware` for your test project to ensure that you test with the correct server and not a custom implementation?

Reply:
Maybe we should actually create a custom middleware for development, that could be added to any UI5 project? Even if we use `preview-middleware` we're limited to the one specific project that we have here and it is always not enough for testing different scenarios.

It would provide an HTML file with control property editor with live reload features.

Reply:
I have reused preview-middleware.
The approach mentioned by Davis can we do it on a different PR. I guess it would need some effort.